package.json

```diff
@@ -17,6 +17,7 @@
     "test": "npm-run-all test:* && vue-cli-service build",
     "test:lintcss": "stylelint '**/*.vue'",
     "test:a11y": "jest --testMatch=\"**/tests/a11y/**\"",
+    "test:cy": "env $(grep -v '^#' .env.example | xargs) start-server-and-test serve http-get://localhost:8080 cypress",
```

Comment:
Is there a way to consolidate this line with line 10 ? 
AFAIK, they're both running the same cypress tests (the integration tests we wrote). or am I missing something ?
and If not, do we need to specify the `headless` and `browser` options here as well ?

Reply:
The command in line 20 is _using_ the command in line 10. I.e., the last `cypress` in line 20 tells it to run `npm run cypress` as soon as it gets a 200 response from the get request to `localhost:8080`.

See the documentation for how `start-server-and-test` works: https://github.com/bahmutov/start-server-and-test#use

Reply:
thank you!
It looks good to me.